x-pack/plugins/cases/server/saved_object_types/migrations/comments.ts

```diff
@@ -8,18 +8,19 @@
 import { mapValues, trimEnd } from 'lodash';
 import { SerializableRecord } from '@kbn/utility-types';
 
-import { LensServerPluginSetup } from '../../../../lens/server';
 import {
   mergeMigrationFunctionMaps,
   MigrateFunction,
   MigrateFunctionsObject,
-} from '../../../../../../src/plugins/kibana_utils/common';
+} from 'src/plugins/kibana_utils/common';
```

Comment:
I think we need the `..` dots. CI is throwing an exception: `[2021-11-30T22:35:08.023+00:00][FATAL][root] Error: Cannot find module 'src/plugins/kibana_utils/common'`